examples/bri-3/prisma/schema.prisma

```diff
@@ -40,3 +41,14 @@ model Transaction {
   signature String
   status Int
 }
+
+model Message {
+  id String @id
+  fromBpiSubjectId String
+  fromBpiSubject BpiSubject @relation(fields: [fromBpiSubjectId], references: [id], name: "fromBpiSubject_fk")
```

Comment:
i think convention is capital first letter according to their docs

Reply:
Fixed